src/components/Wizard/CreateVmWizard/providers/vmwareActions.js

```diff
@@ -0,0 +1,182 @@
+import { get } from 'lodash';
+
+import { SecretModel, V2VVMwareModel } from '../../../../models';
+import { getImportProviderSecretObject, getDefaultSecretName, getV2VVMwareObject } from './vmwareProviderPod';
+
+import {
+  NAMESPACE_KEY,
+  PROVIDER_STATUS_CONNECTING,
+  PROVIDER_STATUS_CONNECTION_FAILED,
+  PROVIDER_VMWARE_URL_KEY,
+  PROVIDER_VMWARE_USER_NAME_KEY,
+  PROVIDER_VMWARE_USER_PWD_AND_CHECK_KEY,
+  PROVIDER_VMWARE_USER_PWD_KEY,
+  PROVIDER_VMWARE_CONNECTION,
+} from '../constants';
+import { CONNECT_TO_NEW_INSTANCE } from '../strings';
+
+export const onVmwareCheckConnection = async (basicSettings, onChange, k8sCreate) => {
+  // Note: any changes to the dialog since issuing the Check-button action till it's finish will be lost due to tight binding of the onFormChange to basicSettings set at promise creation
+  onChange({ status: PROVIDER_STATUS_CONNECTING });
+
+  const url = get(basicSettings[PROVIDER_VMWARE_URL_KEY], 'value');
```

Comment:
`settingsValue` - in the future, we should rename it to `formValue` and probably move to `FormFactory`

Reply:
done